docs/recipes/rsync-deploy.md

```diff
@@ -13,46 +13,47 @@ $ npm install --save-dev gulp-rsync
 ```
 
 ### 2. Create a `rsync config` file
-Add a file to project root `rsync.json`.
 
+Add a file to project root `rsync.json`.
 
-```
+```json
 {
-    "hostname": "hostname or ip",
-    "username": "username",
-    "destination": "folder"
+  "hostname": "hostname or ip",
```

Comment:
According to the [EditorConfig](https://github.com/yeoman/generator-webapp/blob/master/app/templates/editorconfig#L12) file, I switched to 2 spaces for JSON files.